### PR TITLE
Added dbdiagram.io

### LINKED ISSUE #10475: Request: dbdiagram.io
### Brand Name

dbdiagram.io

### Website

https://dbdiagram.io/home

### Popularity Metric

Global Rank on Similarweb: 107,654
https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/3m?webSource=Total&key=dbdiagram.io

### Official Resources for Icon and Color

None

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by nirvikpurkait**
dbdiagramdotio

**Issue:** closes #10475

**Similarweb rank:**
**107,654**

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Icon picked from website's homepage
Color picked from website's css color pallet

### CODE REVIEW

**Review by adamrusted — CHANGES_REQUESTED** [review]
Thanks for this, @nirvikpurkait - and an interesting icon to work with!
I've noted a couple changes to the JSON in the suggestions - but see below for the icon suggestions.

As per our contribution guidelines, I think we might be best including a gap between the different colours - and keeping the icon solid. See the version I've attached below.

image

Below is my path, which you're welcome to use, but we'd need another maintainer to review if you do!

```svg
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>dbdiagram.io</title><path d="M23.985 11.5702V4.575c0-2.52-2.04-4.575-4.575-4.575h-7.58v7.9822l-.5.3616V0H4.59C2.07 0 .015 2.055.015 4.575v14.85C.015 21.945 2.07 24 4.59 24h6.74v-8.3485l.5.3889V24h7.58c2.52 0 4.575-2.055 4.575-4.575v-7.3548H11.83v1.1642l-.5-.3889v-1.7682l.5-.3616v.8544zm-14.6663.3446 5.2 4.0439c.1233-.3649.4603-.6287.8564-.6287h3.3c.51 0 .915.405.915.915v3.3c0 .5101-.405.915-.915.915h-3.3c-.51 0-.915-.405-.915-.915v-2.0927L8.925 13.1478v.3822c0 .51-.42.915-.915.915h-3.3a.909.909 0 0 1-.915-.915v-3.3c0-.51.405-.915.915-.915h3.3c.51 0 .915.405.915.915v.47l5.535-4.0028V4.515c0-.51.42-.915.915-.915h3.3c.51 0 .915.405.915.915v3.3c0 .51-.405.915-.915.915h-3.3c-.3912 0-.7164-.2402-.8486-.5814z"/></svg>
```

File: icons/dbdiagramdotio.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>dbdiagramdotio</title><path d="M4.588 0h6.99v.626h-6.62A4.355 4.355 0 0 0 .61 4.973v14.106a4.356 4.356 0 0 0 4.347 4.347h6.62V24H4.588a4.584 4.584 0 0 1-4.575-4.575V4.575A4.584 4.584 0 0 1 4.588 0Zm7.01 23.427h-.017V24h7.83a4.584 4.584 0 0 0 4.576-4.576V4.575A4.574 4.574 0 0 0 19.411 0H11.58v.626h7.454a4.346 4.346 0 0 1 4.347 4.347v14.106a4.356 4.356 0 0 1-4.347 4.347H11.6Zm7.078-19.824h-3.301a.918.918 0 0 0-.916.915v2.181l-5.534 4.004v-.47a.91.91 0 0 0-.916-.915H4.71a.91.91 0 0 0-.914.916v3.3c0 .51.404.915.914.915h3.3a.918.918 0 0 0 .916-.915v-.38l5.534 4.304v2.09c0 .51.405.915.916.915h3.3a.91.91 0 0 0 .914-.916v-3.3a.91.91 0 0 0-.914-.915h-3.301a.921.921 0 0 0-.868.624l-5.19-4.037 5.204-3.765a.91.91 0 0 0 .854.58h3.3a.91.91 0 0 0 .914-.916v-3.3a.91.91 0 0 0-.914-.915Z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/dbdiagramdotio.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>dbdiagram.io</title><path d="M4.588 0h6.99v.626h-6.62A4.355 4.355 0 0 0 .61 4.973v14.106a4.356 4.356 0 0 0 4.347 4.347h6.62V24H4.588a4.584 4.584 0 0 1-4.575-4.575V4.575A4.584 4.584 0 0 1 4.588 0Zm7.01 23.427h-.017V24h7.83a4.584 4.584 0 0 0 4.576-4.576V4.575A4.574 4.574 0 0 0 19.411 0H11.58v.626h7.454a4.346 4.346 0 0 1 4.347 4.347v14.106a4.356 4.356 0 0 1-4.347 4.347H11.6Zm7.078-19.824h-3.301a.918.918 0 0 0-.916.915v2.181l-5.534 4.004v-.47a.91.91 0 0 0-.916-.915H4.71a.91.91 0 0 0-.914.916v3.3c0 .51.404.915.914.915h3.3a.918.918 0 0 0 .916-.915v-.38l5.534 4.304v2.09c0 .51.405.915.916.915h3.3a.91.91 0 0 0 .914-.916v-3.3a.91.91 0 0 0-.914-.915h-3.301a.921.921 0 0 0-.868.624l-5.19-4.037 5.204-3.765a.91.91 0 0 0 .854.58h3.3a.91.91 0 0 0 .914-.916v-3.3a.91.91 0 0 0-.914-.915Z"/></svg>
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
Nice work, @nirvikpurkait. However, I'm seeing a couple of differences in the corners of the squares with [this one](https://github.com/simple-icons/simple-icons/assets/15157491/05252306-be3f-4378-a284-f6cc769f2cf9) in the top left of the bottom right square being the most pronounced - could you see if you could fic those up, please?

(I checked Adam's path, too, and they're also present there).

**Comment by nirvikpurkait** [comment]
> Nice work, @nirvikpurkait. However, I'm seeing a couple of differences in the corners of the squares with [this one](https://github.com/simple-icons/simple-icons/assets/15157491/05252306-be3f-4378-a284-f6cc769f2cf9) in the top left of the bottom right square being the most pronounced - could you see if you could fic those up, please?
> 
> (I checked Adam's path, too, and they're also present there).

Thanks for letting me know, sorry I didn't see that. After I zoomed in, saw some inconsistency too form the original logo. Fixing is on the way.

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
I'm still seeing some small differences, @nirvikpurkait, most notably now in the outer corners. I gave it a try myself and came up with this unoptimised path, if you'd like to run with it.
```
M4.5898438 0C2.0698463 0 0.015625 2.0542213 0.015625 4.5742188L0.015625 19.425781C0.015625 21.945779 2.0698463 24 4.5898438 24L11.330078 24L11.330078 15.652344L11.830078 16.041016L11.830078 24L19.410156 24C21.930154 24 23.984375 21.945779 23.984375 19.425781L23.984375 12.070312L11.830078 12.070312L11.830078 13.236328L11.330078 12.847656L11.330078 11.076172L11.830078 10.714844L11.830078 11.570312L23.984375 11.570312L23.984375 4.5742188C23.984375 2.0542213 21.945154 5.9211835e-16 19.410156 0L11.830078 0L11.830078 7.9804688L11.330078 8.34375L11.330078 0L4.5898438 0zM15.375 3.5996094L18.675781 3.5996094C19.185781 3.5996094 19.589844 4.0056265 19.589844 4.515625L19.589844 7.8144531C19.589844 8.3244516 19.185781 8.7304688 18.675781 8.7304688L15.375 8.7304688C14.983892 8.7304687 14.655741 8.4914526 14.523438 8.1503906L9.3183594 11.916016L14.509766 15.951172C14.634004 15.587932 14.980073 15.330078 15.375 15.330078L18.675781 15.330078C19.185781 15.330078 19.589844 15.734141 19.589844 16.244141L19.589844 19.544922C19.589844 20.054919 19.185781 20.460938 18.675781 20.460938L15.375 20.460938C14.865003 20.460938 14.460938 20.054919 14.460938 19.544922L14.460938 17.453125L8.9257812 13.148438L8.9257812 13.529297C8.9257812 14.039294 8.5047641 14.445312 8.0097656 14.445312L4.7109375 14.445312C4.200939 14.445313 3.7949219 14.039294 3.7949219 13.529297L3.7949219 10.230469C3.7949219 9.7204699 4.200939 9.3144531 4.7109375 9.3144531L8.0097656 9.3144531C8.5197641 9.3144531 8.9257813 9.7204699 8.9257812 10.230469L8.9257812 10.699219L14.460938 6.6953125L14.460938 4.515625C14.460938 4.0056265 14.88 3.5996094 15.375 3.5996094z
```

**Pull request closed without merging**